src/special/commitizen.js

```diff
@@ -0,0 +1,22 @@
+import path from 'path';
+
+export default function parseCommitizen(content, filePath, deps, rootDir) {
+  const packageJsonPath = path.resolve(rootDir, 'package.json');
+  const resolvedFilePath = path.resolve(filePath);
+
+  if (resolvedFilePath === packageJsonPath) {
+    const metadata = JSON.parse(content);
+
+    if (metadata.config && metadata.config.commitizen && metadata.config.commitizen.path) {
+      const commitizenPath = metadata.config.commitizen.path;
+
+      if (commitizenPath.startsWith('./node_modules/')) {
+        return [commitizenPath.substring(15)];
+      } else if (!commitizenPath.startsWith('./')) {
```

Comment:
Besides these two cases, the user may write full OS path inside the box too. My above code snippet will handle that.